EX2Kit/Macros/EX2Macros.h

```diff
@@ -29,11 +29,7 @@
 #define IS_TV() (UI_USER_INTERFACE_IDIOM() == UIUserInterfaceIdiomTV)
 #endif
 
-#ifdef IMSG
-#define IS_IMSG() (true)
-#else
 #define IS_IMSG() (false)
```

Comment:
why not remove this line too?

Reply:
Ah my bad, I forgot we're removing everything related to IMSG